File: customview/src/main/java/dinson/customview/adapter/_004LeftAdapter.java
package dinson.customview.adapter;

import android.content.Context;

import java.util.List;

import dinson.customview.R;
import dinson.customview.entity.MonsterHunter.DataBean;
import dinson.customview.weight.recycleview.CommonAdapter;
import dinson.customview.weight.recycleview.CommonViewHolder;

/**
 * @author Dinson - 2017/7/21
 */
public class _004LeftAdapter extends CommonAdapter<DataBean> {


    public _004LeftAdapter(Context context, List<DataBean> dataList) {
        super(context, dataList);
    }

    @Override
    public int getLayoutId(int viewType) {
        return R.layout.item_004_left;
    }

    @Override
    public void convert(CommonViewHolder holder, DataBean dataBean, int position) {
        holder.setTvText(R.id.tv_name, dataBean.getFamily());
    }
}

File: customview/src/main/java/dinson/customview/adapter/_004LeftAdapter.kt
package dinson.customview.adapter

import android.content.Context

import dinson.customview.R
import dinson.customview.entity.MonsterHunter.DataBean
import dinson.customview.weight.recycleview.CommonAdapter
import dinson.customview.weight.recycleview.CommonViewHolder

/**
 * @author Dinson - 2017/7/21
 */
class _004LeftAdapter(context: Context, dataList: List<DataBean>)
    : CommonAdapter<DataBean>(context, dataList) {

    override fun getLayoutId(viewType: Int)=  R.layout.item_004_left

    override fun convert(holder: CommonViewHolder, dataBean: DataBean, position: Int) {
        holder.setTvText(R.id.tvName, dataBean.family)
    }
}
